### PR TITLE
FuzzAI: First set of files

### PR DESCRIPTION
**Pull request by psiinon**
## Overview
A new set of fuzzing files aimed at testing LLMs

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: addOns/fuzzai/fuzzai.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+plugins {
+    eclipse
+}
+
+eclipse {
+    classpath {
+        // Prevent compilation of zapHomeFiles.
+        sourceSets = listOf()
+    }
+}
```
Comment (thc202) [inline addOns/fuzzai/fuzzai.gradle.kts:12]: Could be removed for this one.

File: addOns/fuzzai/src/main/javahelp/help/map.jhm [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+    <mapID target="fuzzai" url="contents/fuzzai.html" />
```
Comment (thc202) [inline addOns/fuzzai/src/main/javahelp/help/map.jhm:7]: Could use the `addon.` prefix for the keys.

**Comment by psiinon** [comment]
Hopefully fixed those :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**